src/glib/base/shash.h

```diff
@@ -1005,6 +1005,8 @@ class THashSetKey{
   void SaveXml(TSOut& SOut, const TStr& Nm) const {
     XSaveHd(Nm); XSave(Key); }
 
+  int GetMemUsed() const { return sizeof(TKey) + 2*sizeof(int); }
```

Comment:
Could you use new more robust way for calculating size, same as in TPair (https://github.com/qminer/qminer/blob/master/src/glib/base/ds.h#L65)